osidb/serializer.py

```diff
@@ -919,19 +919,40 @@ class Meta:
         abstract = True
 
 
-class AffectCVSSSerializer(
+class AbstractCVSSSerializer(
     ACLMixinSerializer,
     AlertMixinSerializer,
     BugzillaSyncMixinSerializer,
     IncludeExcludeFieldsMixin,
     TrackingMixinSerializer,
+):
+    """Abstract serializer for FlawCVSS and AffectCVSS serializer"""
+
+    cvss_version = serializers.ChoiceField(
+        choices=FlawCVSS.CVSSVersion, source="version"
```

Comment:
Please use `CVSS.CVSSVersion` instead of `FlawCVSS.CVSSVersion`

Reply:
Thanks, fixed.